blog/2021/07/general-survey.md

```diff
@@ -153,54 +157,54 @@ staggering 95% of packages in the ecosystem are set up to use any continuous
 integration service, with GitHub Action being the most popular service (80%),
 followed by Travis (42%) and AppVeyor (13%).
 
-![Distributions of continuois integration services](/_assets/blog/2021-07-general-survey/fraction-ci.png)
+![Distributions of continuois integration services](/assets/blog/2021-07-general-survey/fraction-ci.png)
 
 We can take a look at the popularity of CI services over time: a common trend is
-that GitHub Actions is taking over the couple Travis and Appveyor, which used to
-be the main choice for Julia users to test their package on the most popular
+that GitHub Actions is taking over from Travis and Appveyor, which used to
+be the main choice for Julia users to test their packages on the most popular
 operating systems (Linux, macOS, Windows).  The decreasing trend for Travis has
 been exacerbated last year when changes to their conditions made this service
 less favourable to be used by open-source projects.
 
-![Trend of popularity of CI services over time](/_assets/blog/2021-07-general-survey/trend-ci.png)
+![Trend of popularity of CI services over time](/assets/blog/2021-07-general-survey/trend-ci.png)
 
-## Contibutors
+## Contributors
 
 As of 2021-06-27, at least 4931 different users contributed to Julia packages
 hosted on GitHub, about 250 more compared to two months before.  Note that these
 figures are conservative as they don't include contributors identified as
 anonymous by GitHub API (for example because they committed changes using an
 email address not registered in any GitHub account), or any contributions to
-packages hosted on different websites, like Gitlab.
+packages hosted on different websites, like GitLab.
 
-![Trend of numbers of contributors](/_assets/blog/2021-07-general-survey/trend-contributors.png)
+![Trend of numbers of contributors](/assets/blog/2021-07-general-survey/trend-contributors.png)
 
 The distribution of number of contributors to Julia packages presents a grim
```

Comment:
I thought maybe "grim" was a bit too negative here but wasn't sure what else to say